src/routes/login/+page.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import Login from "./login.svelte";
+
+  let username:string;
+  let password:string;
+
+  const handleLogin = async () => {
+    try {
+      const response = await fetch('/auth/login', {
+        method: 'POST',
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          username,
+          password_hash: hash(password)
+        })
+      });
+      
+      const data = await response.json();
+      if (response.ok) {
+        // change later if home page changes 
+        console.log(response.json);
+        () => window.location.href = '/';
+      } else {
+        console.error('Login failed:', data.message);
+      }
+    } catch (error) {
+      console.error('Error during Login:', error)
+    }
+  }
+
+
+  // No Hashing yet, implement
+  function hash(password:string):string {
+    return password;
+  }
+</script>
+
+<Login on:click={handleLogin} {username} {password}/>
```

Comment:
I would consider breaking the `Login.svelte` into separate and smaller components. This is purely for readability as these components would not be reused. I would probably also move the code responsible for the login handling into the component hosting the two inputs to avoid using external/global variables. However your abstraction might be a technique I have not seen yet and might be completely fine.